test-harness/helpers.ts

```diff
@@ -1,41 +1,19 @@
-import * as  fs from 'fs';
-import { join } from 'path';
-import fetch from 'node-fetch';
+export function parseSpecFile(spec: string): any {
+  const regex = /====(server|test|spec|command|expect)====\n(([^=])*)/gi
```

Comment:
```suggestion
  const regex = /====(server|test|spec|command|expect)====\r?\n(([^=])*)/gi
```
Let's add ca arriage return here for all the Windows folks :)

Moreover, I'd move it outside the function and do the following here
```suggestion
  regex.lastIndex = 0;
```
It's a test though, so you can leave it as is.